src/app/pages/demo/carousel-lcp/carousel-lcp.html

```diff
@@ -0,0 +1,56 @@
+<div class="container">
+
+  <h2 class="link clickable">
+    <a class="link-title" href="https://github.com/Ks89/angular-modal-gallery">Carousel lcp</a>
+  </h2>
+  <p>Carousel lcp demo.</p>
```

Comment:
Example is ok, but I think It needs some more information. For example, what does it mean LCP? I Suggest to write at least a small phrase with a link to a website with some official information about the topic.

Reply:
we can link that: https://web.dev/lcp/

Reply:
good idea